imagedephi/base_rules.yaml

```diff
@@ -82,3 +82,615 @@ rules:
     - tag_name: Predictor
       method: keep
       type: metadata
+    - tag_name: Software
+      method: delete
+      type: metadata
+    - tag_name: ReferenceBlackWhite
+      method: keep
+      type: metadata
+    - tag_name: GPSVersionID
+      method: delete
+      type: metadata
+    - tag_name: '11'
+      method: delete
+      type: metadata
+    - tag_name: GPSMapDatum
+      method: delete
+      type: metadata
+    - tag_name: GPSLatitudeRef
+      method: delete
+      type: metadata
+    - tag_name: OldSubfileType
+      method: delete
+      type: metadata
+    - tag_name: Threshholding
+      method: delete
+      type: metadata
+    - tag_name: FillOrder
+      method: delete
+      type: metadata
+    - tag_name: DocumentName
+      method: delete
+      type: metadata
+    - tag_name: Make
+      method: delete
+      type: metadata
+    - tag_name: Model
+      method: delete
+      type: metadata
+    - tag_name: MinSampleValue
+      method: delete
+      type: metadata
+    - tag_name: MaxSampleValue
+      method: delete
+      type: metadata
+    - tag_name: PageName
+      method: delete
+      type: metadata
+    - tag_name: Xposition
```

Comment:
Don't delete XPosition or Yposition (again, we'd ideally check that these are numerical values).